### ISSUE TITLE
SyntaxError on literal with \' and "

### ISSUE THREAD

**Issue by snarfed** [ikamensh/flynt#114]
First off, thank you for flynt, it's great!

I'm seeing this failure on a literal with both escaped and unescaped quotes. I assume it's not expected?

`x.py`:
```py
var = 'baz'
'foo " %s \' bar' % var
```

```sh
$ flynt -v x.py
Running flynt v.0.69
Failed to parse transformed code 'f'bazfoo " {var} ' bar'' given original ''baz''foo " %s \' bar' % var'
invalid syntax (<unknown>, line 1)
Traceback (most recent call last):
  File "/Users/ryan/src/oauth-dropins/local/lib/python3.9/site-packages/flynt/transform/transform.py", line 49, in transform_chunk
    ast.parse(new_code)
  File "/opt/homebrew/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    f'bazfoo " {var} ' bar'
                       ^
SyntaxError: invalid syntax
fstringifying /Users/ryan/x.py...no change
...
```

I'd expect it to convert to:

```
var = 'baz'
f'foo " baz \' bar'
```

Interestingly, if I switch the quote characters like so:

```py
var = 'baz'
"foo ' %s \" bar" % var
```

flynt still fails to convert it, but doesn't show an exception:

```sh
$ flynt -v x.py
Running flynt v.0.69
Faulty result during conversion on /Users/ryan/x.py: statement count has changed, which is not intended - skipping.
fstringifying /Users/ryan/x.py...no change
...
```

Obviously not critical. Thanks again!

**Comment by snarfed** [comment]
(#44 is maybe loosely related, but probably not the same since it was fixed a while ago)

**Comment by ikamensh** [comment]
Fixed in v0.74. Thanks for the bug report
